File: app/src/androidTest/java/com/shen/xi/android/tut/test/AndroidTestSuit.java
package com.shen.xi.android.tut.test;


import android.test.suitebuilder.TestSuiteBuilder;

import junit.framework.Test;

public class AndroidTestSuit {
    public static Test suite() {
        return new TestSuiteBuilder(AndroidTestSuit.class)
                .includeAllPackagesUnderHere()
                .build();
    }
}

File: app/src/androidTest/scala/com/shen/xi/android/tut/test/AndroidTestSuit.scala
package com.shen.xi.android.tut.test


import android.test.suitebuilder.TestSuiteBuilder


object AndroidTestSuit {

  def suite() =
    new TestSuiteBuilder(AndroidTestSuit.getClass)
      .includeAllPackagesUnderHere()
      .build()

}
